src/main/java/de/metas/ui/web/view/DefaultViewsRepositoryStorage.java

```diff
@@ -36,12 +38,14 @@
 // NOTE: don't add it to spring context! i.e. don't annotate it with @Component or similar
 public final class DefaultViewsRepositoryStorage implements IViewsIndexStorage
 {
+	private static transient final Logger logger = LogManager.getLogger(DefaultViewsRepositoryStorage.class);
+
 	private final Cache<ViewId, IView> views;
 
-	public DefaultViewsRepositoryStorage(final int viewExpirationTimeoutInHours)
+	public DefaultViewsRepositoryStorage(final long viewExpirationTimeoutInMinutes)
```

Comment:
@TheBestPessimist  @metas-ts  we shall avoid using primitive (and meaningless)  types.
In this case, Duration would be suitable.

Reply:
THere's no way in java 8 to transform a Duration -> Timeunit. 
I wanted to use duration in the first place, but didn't see a good way. More details: https://stackoverflow.com/questions/26581708/converting-timeunit-to-chronounit